main/src/ca/uwaterloo/flix/language/phase/Namer.scala

```diff
@@ -1762,6 +1764,21 @@ object Namer {
       }
   }
 
+  /**
+    * Translates the given weeded JvmMethod to a named JvmMethod.
+    */
+  private def visitJvmMethod(method: WeededAst.JvmMethod, uenv: UseEnv, tenv: Map[String, Symbol.UnkindedTypeVarSym])(implicit flix: Flix): Validation[NamedAst.JvmMethod, NameError] = {
+    flatMapN(traverse(method.fparams)(visitFormalParam(_, uenv, tenv))) {
+      case fparams =>
+        val exp = visitExp(method.exp, getVarEnv(fparams), uenv, tenv)
```

Comment:
Thus here you would take the enviroment `env` and extend it with the result of getVarEnv. `(++)`